Tests/AwesomeAssertions.Specs/Primitives/StringAssertionSpecs.Match.cs

```diff
@@ -167,5 +167,20 @@ public void When_a_string_is_negatively_matched_against_an_empty_string_it_shoul
                     "Cannot match string against an empty string. Provide a wildcard pattern or use the NotBeEmpty method.*")
                 .WithParameterName("wildcardPattern");
         }
+
+        [Fact]
+        public void Null_does_not_negatively_match_to_any_pattern()
+        {
+            // Arrange
+            string subject = null;
+
+            // Act
+            Action act = () => subject.Should().NotMatch("*", "because that is the {0}", "current behavior");
```

Comment:
```suggestion
            Action act = () => subject.Should().NotMatch("*", "failure {0}", "message");
```
I would recommend to always use the same pattern to verify the because formatting.
My suggestion applies to the pattern I've seen recently. But, maybe we could define a more readable pattern!?
You may ignore this suggestion: I'll make a separate PR with defining a coding guide and its updates.

Reply:
Yes, would be nice to have the same pattern. Currently we already have several different ways. I guess FA changed at some point, but didn't adapt all the tests. They say, that such changes should only be done "on the way", when doing other changes anyway.
But I think, that this leads to unnecessary discussions, because it isn't apparent, which one is "the right way". This was the reason for e.g #191 . We have a rule set at https://github.com/AwesomeAssertions/AwesomeAssertions/wiki/Design-Guidelines